modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/SingleValueSetter.java

```diff
@@ -45,6 +46,6 @@ public SingleValueSetter(ParameterModel parameterModel, Field field)
     @Override
     public void set(Object target, ResolverSetResult resolverSetResult)
     {
-        fieldSetter.set(target, resolverSetResult.get(parameterModel));
+        Optional.ofNullable(resolverSetResult.get(parameterModel)).ifPresent(v -> fieldSetter.set(target, v));
```

Comment:
yes, but that's something you can do with a simple if.

Instead you're:

* Instantiating an optional (which ends up checking null)
* Instantiating a lamba (which gets instantiated even if the value is indeed null)

Again, this would be just fine if you were planning to externalize the value. But since this is a private piece of code, simply check null with an if